.travis.yml

```diff
@@ -9,10 +9,14 @@ env:
   - PANDAS=0.21.0
   - PANDAS=0.23.4
   - PANDAS=latest
+services:
```

Comment:
❤️

Reply:
Thought for later: we should add MySQL next, because it's pretty easy in travis and will help verify our hope/claim that SQLAlchemy is a universal translator among SQL engines.

This'll require a bunch of type-declaration blobs in test fixtures...

Reply:
👍